docs/process/first-responder.md

```diff
@@ -12,7 +12,8 @@ Each rotation is a week long. While first responder your primary duties are:
 1. Triage issues.
     * Troubleshoot or follow up on troubleshooting started by the previous first responder. The current first responder should always bear responsibility for pushing troubleshooting forward, unless another team member has explicitly taken ownership.
     * Ensure issues are labeled accurately.
-    * Review the [`more-information-needed`](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+label%3Amore-information-needed+sort%3Aupdated-asc) and [`needs-reproduction`](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+sort%3Aupdated-asc+label%3Aneeds-reproduction) issues and close any that have gone 2 weeks with no new activity.
+    * Review issues labelled [`needs-reproduction`](https://github.com/desktop/desktop/issues?q=is%3Aopen+is%3Aissue+sort%3Aupdated-asc+label%3Aneeds-reproduction) and close any that have gone 2 weeks with no new activity after the last question by an official maintainer.
```

Comment:
This label is now `reviewer-needs-to-reproduce`

Reply:
I've been using "reviewer" when referring to a person who isn't the submitter who is looking at issues. Not sure how strongly we feel on either phrase, but "official maintainer" feels too formal for my liking.